doc/source/cluster/cloud.rst

```diff
@@ -45,10 +45,15 @@ Ray with cloud providers
         # Tear down the cluster.
         $ ray down ray/python/ray/autoscaler/aws/example-full.yaml
 
+    .. note::
+
+        See :ref:`aws-cluster` for additional AWS integrations and optimizations.
+
+        Refer to the `ray/python/ray/autoscaler/aws/example-gpu-docker.yaml <https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/aws/example-gpu-docker.yaml>`__ cluster config file to setup a GPU cluster on AWS.
```

Comment:
```suggestion
```

Seems a bit weird that we're providing a docker specific example here, redirecting them to the aws cluster page should be enough.